settling_probe.py

```diff
@@ -4,12 +4,12 @@
 # Copyright (C) 2023 Mitko Haralanov <voidtrance@gmail.com>
 #
 # This file may be distributed under the terms of the GNU GPLv3 license.
-import probe
+from .probe import probe as probe_PrinterProbe
```

Comment:
I don't understand this statement!
From the module `probe` you are importing the object `probe` with the name `probe_PrinterProbe`. However, the object (class) `PrinterProbe` is never imported. How does this work?

What kind of object is `probe_PrinterProbe`?

Reply:
Under the python3 virtual environment for Klipper (not python2) with your current ``settling_probe.py`` file I get the following error and the Printer will not boot up:

From the log file:

=======================
Unhandled exception during connect
Traceback (most recent call last):
  File "/home/pi/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/pi/klipper/klippy/klippy.py", line 141, in _read_config
    self.load_object(config, section_config.get_name(), None)
  File "/home/pi/klipper/klippy/klippy.py", line 121, in load_object
    mod = importlib.import_module('extras.' + module_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/pi/klipper/klippy/extras/settling_probe.py", line 7, in <module>
    import probe
ModuleNotFoundError: No module named 'probe'
[klippy.log](https://github.com/voidtrance/voron-klipper-extensions/files/10559379/klippy.log)

Reply:
When I change the import statement to 
```python
from .probe import probe
```
I get this error in the log file:

=======================
Unhandled exception during connect
Traceback (most recent call last):
  File "/home/pi/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/pi/klipper/klippy/klippy.py", line 141, in _read_config
    self.load_object(config, section_config.get_name(), None)
  File "/home/pi/klipper/klippy/klippy.py", line 121, in load_object
    mod = importlib.import_module('extras.' + module_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/pi/klipper/klippy/extras/settling_probe.py", line 7, in <module>
    from .probe import probe
ImportError: cannot import name 'probe' from 'extras.probe' (/home/pi/klipper/klippy/extras/probe.py)
webhooks client 547824126224: New connection
webhooks client 547824126224: Client info {'program': 'Moonraker', 'version': 'v0.7.1-831-g61d3182'}

[klippy.log](https://github.com/voidtrance/voron-klipper-extensions/files/10559414/klippy.log)

Reply:
If I use the following import statement, I still get an error:

```python
from .probe import probe as probe_OBJ
```
and then make the changes from ``probe.`` to ``probe_OBJ.``

The error is as follows:
=======================
Unhandled exception during connect
Traceback (most recent call last):
  File "/home/pi/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/pi/klipper/klippy/klippy.py", line 141, in _read_config
    self.load_object(config, section_config.get_name(), None)
  File "/home/pi/klipper/klippy/klippy.py", line 121, in load_object
    mod = importlib.import_module('extras.' + module_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/pi/klipper/klippy/extras/settling_probe.py", line 7, in <module>
    from .probe import probe as probe_OBJ
ImportError: cannot import name 'probe' from 'extras.probe' (/home/pi/klipper/klippy/extras/probe.py)
webhooks client 547695266544: New connection
webhooks client 547695266544: Client info {'program': 'Moonraker', 'version': 'v0.7.1-831-g61d3182'}
[klippy.log](https://github.com/voidtrance/voron-klipper-extensions/files/10559489/klippy.log)

Reply:
Here is the message I was getting when the import statement is as follows:

```
import probe
```
Log error message when using python3 virtual environment with Klipper
```
Unhandled exception during connect
Traceback (most recent call last):
  File "/home/pi/klipper/klippy/klippy.py", line 175, in _connect
    self._read_config()
  File "/home/pi/klipper/klippy/klippy.py", line 141, in _read_config
    self.load_object(config, section_config.get_name(), None)
  File "/home/pi/klipper/klippy/klippy.py", line 121, in load_object
    mod = importlib.import_module('extras.' + module_name)
  File "/usr/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 790, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/home/pi/klipper/klippy/extras/settling_probe.py", line 7, in <module>
    import probe
ModuleNotFoundError: No module named 'probe'
webhooks client 548178641728: New connection
webhooks client 548178641728: Client info {'program': 'Moonraker', 'version': 'v0.7.1-831-g61d3182'}
```

Reply:
New commit has been made, using the following import statement:

```
from .probe import PrinterProbe as PrinterProbe
```
I hope this resolve this issue